R/LSbinomialtesting.R

```diff
@@ -907,6 +907,12 @@ LSbinomialtesting   <- function(jaspResults, dataset, options, state = NULL) {
 }
 .tablePredictions2LS                    <- function(jaspResults, data, ready, options, type = c("Prior", "Posterior")) {
 
+  if(!ready["data"]) {
```

Comment:
I think this creates a slightly different behavior than desired --- it passes an empty data set, rather than creating an "empty" waiting plot/table. I would prefer a table is initiated and then a return statement is issued if the data are not ready (instead of overwriting data early on).

(I didn't check the source code whether it's possible --- let me know if not)